.rubocop_todo.yml

```diff
@@ -7,12 +7,12 @@
 
 # Offense count: 8
 Metrics/AbcSize:
-  Max: 51
+  Max: 53
```

Comment:
Personally I don't care as much, I think it's a good flag and if it makes sense, I would not make it a barrier.